### PR TITLE
Resolves #41 - Added support for the fully qualified classname

### LINKED ISSUE #41: DataGrid should accept the fully classified classname
Currently the Datarid only allows text alias of the DataGridFieldType's.

I propose to deprecate this usage of alias, and move to fully classified classnames.

```
->addField('field_name', 'text');
```

becomes

```
->addField('field_name', DatagridFieldTypeText::class);
```

### PR DESCRIPTION
**Pull request by pinoniq**
Added support for the fully qualified classname in favor of text alias

### CODE REVIEW

File: doc/datagrid.md [hunk 1]
```diff
@@ -143,13 +143,13 @@ If a callback is defined it must return a value that can be converted to boolean
 
 ``` php
 ->addField(
-    'comments', 
-    'collection', 
+    'comments',
+    'collection',
```
Comment (wanjee) [inline doc/datagrid.md]: Should be changed to FQN as well

File: doc/datagrid.md [hunk 2]
```diff
@@ -175,14 +175,14 @@ If a callback is defined it must return an array or an object that implements th
 
 ``` php
 ->addField(
-    'published', 
-    'date', 
+    'published',
+    'date',
```
Comment (wanjee) [inline doc/datagrid.md]: Should probably be changed to FQN as well

File: doc/datagrid.md [hunk 3]
```diff
@@ -198,14 +198,14 @@ If a callback is defined it must return a [Datetime](http://php.net/manual/en/cl
 
 ``` php
 ->addField(
-    'image', 
-    'image', 
+    'image',
+    'image',
```
Comment (wanjee) [inline doc/datagrid.md]: Should probably be changed to FQN as well

File: doc/datagrid.md [hunk 4]
```diff
@@ -222,13 +222,13 @@ If a callback is defined it must return a string: the image name.
 
 ``` php
 ->addField(
-    'id', 
-    'text', 
+    'id',
+    'text',
```
Comment (wanjee) [inline doc/datagrid.md]: Should probably be changed to FQN as well

File: doc/datagrid.md [hunk 5]
```diff
@@ -254,15 +254,15 @@ If a callback is defined it must return a string.
 
 ``` php
 ->addField(
-    'id', 
-    'link', 
+    'id',
+    'link',
```
Comment (wanjee) [inline doc/datagrid.md]: Should probably be changed to FQN as well

File: src/Datagrid/DatagridManager.php [hunk 6]
```diff
@@ -17,25 +17,39 @@ class DatagridManager
...
+    /**
      * @param \Wanjee\Shuwee\AdminBundle\Datagrid\Field\Type\DatagridFieldTypeInterface $type
+     * @param string $alias
      */
-    public function registerType($alias, DatagridFieldTypeInterface $type)
+    public function registerType(DatagridFieldTypeInterface $type, $alias = null)
     {
-        $this->types[$alias] = $type;
+        $this->types[get_class($type)] = $type;
+        $this->alias[$alias] = get_class($type);
```
Comment (pinoniq) [inline src/Datagrid/DatagridManager.php]: I should only add an alias if one is defined

**Pull request merged**